docs/changes.rst

```diff
@@ -39,6 +39,9 @@ Released: not yet
 
 * Test: Added and fixed profile definitions for end2end tests. (Issue #1714)
 
+* Test: Fixed Appveyor CI setup for UNIX environments under Windows
```

Comment:
I think the the common acronym for this is WSL (Windows Subsystem for Linux). Probably more people will understand if we use that terminology.

Reply:
WSL is something different than what CygWin and MSYS2 provide: https://en.wikipedia.org/wiki/Windows_Subsystem_for_Linux. Neither CygWin nor MSYS2 register itself as a WSL, nor do they intend to serve as a runtime compatibility layer for running Linux executables in WLF format. Instead, they serve mostly as a development environment for executables to be run on native Windows (in addition to be run in their respective shells).

Reply:
Karl agreed in the slack channel. DONE.